partials/header.hbs

```diff
@@ -53,6 +52,14 @@
                     rel="noopener noreferrer" aria-label="LinkedIn">
                     {{> icons/linkedin}}
                 </a>
+                <div class="dark-light">
```

Comment:
When dark is enabled, the sub-menu goes all white

<img width="903" alt="image" src="https://user-images.githubusercontent.com/792287/155053032-c703879f-4557-4d85-99e7-cd4875f7cc6d.png">